src/main/scala/org/hatdex/dex/api/json/DexJsonFormats.scala

```diff
@@ -17,14 +17,14 @@ trait DexJsonFormats extends HatJsonFormats {
   import play.api.libs.json.JodaWrites._
   import play.api.libs.json.JodaReads._
 
-  implicit val offerHatFormat            = Json.format[OfferHat]
-  implicit val offerclaimFormat          = Json.format[OfferClaim]
-  implicit val offerHatCredentialsFormat = Json.format[OfferHatCredentials]
-  implicit val offerClaimsInfoFormat     = Json.format[OfferClaimsInfo]
+  implicit val offerHatFormat: OFormat[OfferHat]            = Json.format[OfferHat]
```

Comment:
I see the formatting is wrong here. I noticed that the `sbt scalafmt` is not running as part of the sbt build command. I need to manually call it before each PR. Ben said he will take a look. Just FYI

Reply:
It's the ordering of scalafix and scalafmt.